<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>681</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MADE Outer Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../08.-Violet-Blue.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>470</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../ftlogo2s.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MADE Outer Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Here are a few recommendations</string>
    </property>
   </widget>
   <widget class="QLabel" name="subtext">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MADE Outer Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Fine-Tune your music.</string>
    </property>
   </widget>
   <widget class="QLabel" name="FindTune">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Basement Grotesque</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FindTune</string>
    </property>
   </widget>
   <widget class="QLabel" name="img1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPixmap{
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img1_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPixmap{
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img1_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPixmap{
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img1_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPixmap{
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img1_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPixmap{
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img1_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPixmap{
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MADE Outer Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Song 1</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MADE Outer Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Song 1</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MADE Outer Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Song 1</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MADE Outer Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Song 1</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MADE Outer Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Song 1</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MADE Outer Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Song 1</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
